dev/com.ibm.jbatch.internal.ee10/bnd.bnd

```diff
@@ -0,0 +1,48 @@
+#*******************************************************************************
+# Copyright (c) 2022 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+
+-sub: *.bnd
```

Comment:
No reason to do a sub bundle if there is only 1 bundle.  Just put what is in the original.bnd into this bnd file.